utils/sub8_msgs/msg/PoseTwist.msg

```diff
@@ -0,0 +1,3 @@
+std_msgs/Header header
```

Comment:
In the future, the controller will *only* take trajectory messages, so that we can set frames. It would be redundant to force each waypoint message to carry a full header.